EuclideanGeometry/Foundation/Construction/Triangle/IsocelesTriangle.lean

```diff
@@ -1,6 +1,19 @@
 import EuclideanGeometry.Foundation.Axiom.Triangle.Basic
 import EuclideanGeometry.Foundation.Axiom.Triangle.Basic_ex
 import EuclideanGeometry.Foundation.Axiom.Triangle.Trigonometric
+import EuclideanGeometry.Foundation.Axiom.Triangle.Congruence
+import Mathlib.Data.Real.Basic
+import Mathlib.Analysis.SpecialFunctions.Trigonometric.Basic
+
+--文件外
+-- 增加了edge_eq_edge_of_flip_vertices等四个关于 perm,flip的边,角相等的定理，在Triangle.Basic.ex中
+-- Triangle.Basic判断clock的地方：clock_of_neg_angle 结论从is_cclock 改为 ¬ ... is_cclock
+-- 判断三角形反全等的地方：acongr_of_AAS，acongr_of_ASA；的结论从全等改为反全等
+--文件内
+-- 第一题is_isoceles_tri_iff_ang_eq_ang_of_nd_tri把angle₂=-angle₃ 删去负号
+-- 最后三题regular_tri_of_isoceles_tri_of_fst_ang_eq_sixty_deg 等，加上了isisoceles 的条件
+-- ang_eq_sixty_deg_of_cclock_regular_tri 加上了等边三角形的条件
+-- 填上了所有sorry
```

Comment:
Please delete comments that are not for common readers but for you and me only from the file.